platforms/dir.proj

```diff
@@ -17,6 +17,9 @@
     <Project Include="xamarin.mac/*.csproj">
       <TargetGroup>xamarin.mac</TargetGroup>
     </Project>
+    <Project Include="xamarin.mac/Facades/*.csproj">
```

Comment:
It's how we organize them in Mono. I can change it if you want, but I think it helps keeping track of what is a facade and what is a regular assembly.